src/AltinnCore/Templates/SingleDataTask.bpmn

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
in which context is this bpmn file used?

Reply:
This is the same file as process.bpmn, I expect we will have multiple template process models and when app-developer select one it will be called process.bpmn